frontend/src/bundles/common/components/sidebar/sidebar.tsx

```diff
@@ -8,7 +8,7 @@ import { getValidClassNames } from '../../helpers/helpers.js';
 import { useAppSelector, useCallback, useState } from '../../hooks/hooks.js';
 import styles from './styles.module.scss';
 
-const menuItems = [
+const generalMenu = [
```

Comment:
In changed name follow the recommendations
<img width="787" alt="image" src="https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/60103912/d69dac2b-bbec-4ded-81ab-bbb49271610d">